test/test_testing.py

```diff
@@ -1818,6 +1818,29 @@ def test_no_warning_on_import(self) -> None:
             cwd=os.path.dirname(os.path.realpath(__file__)),).decode("utf-8")
         self.assertEquals(out, "")
 
+    @unittest.skipIf(IS_WINDOWS, "importing torch+CUDA on CPU results in warning")
```

Comment:
No need to skip since you don't check for warnings here?

Reply:
The warning is going to change the stdout/stderr output, right? Then that affects the string we do a comparison with